hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/endpoint/JerseyClientFactory.java

```diff
@@ -16,7 +16,7 @@ public static ClientConfig createConfig() {
         ClientConfig clientConfig = new ClientConfig();
         clientConfig.property(ClientProperties.ASYNC_THREADPOOL_SIZE, 10);
         clientConfig.property(ClientProperties.CONNECT_TIMEOUT, 5000);
-        clientConfig.property(ClientProperties.READ_TIMEOUT, 5000);
+        clientConfig.property(ClientProperties.READ_TIMEOUT, 20000);
```

Comment:
Why? :)

Reply:
because hermes-management is now waiting for hermes-consumers until they move offsets during retransmission. We have tests for this, therfore we had to increase timeout here.